client/me/notification-settings/settings-form/constants.js

```diff
@@ -1,4 +1,4 @@
 export const NOTIFICATIONS_EXCEPTIONS = {
-	timeline: [ 'draft_post_prompt' ],
+	timeline: [ 'draft_post_prompt', 'recommended_blog' ],
```

Comment:
This removes it from the masterbar notifications... I guess we can add this still even though we don't have a masterbar notification yet? Thoughts?

Reply:
Adding the exceptions makes sense to me 👍